Fetch more keys -  review messages (#2466)
cyberark/conjur: CHANGELOG.md

## [Unreleased]

### Added

> -  Added an ability to fetch signing keys from JWKS endpoints are using self-signed

Reviewer: ```suggestion
-  Added the ability to fetch signing keys from JWKS endpoints that use a self-signed
```

>    certificate or certificate signed by 3rd party CA to JWT generic vendor configuration

Reviewer: ```suggestion
   certificate or a certificate signed by a third-party CA for JWT generic vendor configuration
```

  ([#2462](https://github.com/cyberark/conjur/pull/2462)
  [#2461](https://github.com/cyberark/conjur/pull/2461)
  [#2456](https://github.com/cyberark/conjur/pull/2456)
  [#2455](https://github.com/cyberark/conjur/pull/2455)
  [#2457](https://github.com/cyberark/conjur/pull/2457)
  [#2452](https://github.com/cyberark/conjur/pull/2452)
  [#2437](https://github.com/cyberark/conjur/pull/2437))

> -  Added an ability to JWT generic vendor configuration to receive signing keys for

Reviewer: ```suggestion
-  Added the ability for JWT generic vendor configuration to receive signing keys for
```
